src/client/pages/AssessmentSection/DataTable/Table/Row/RowData/Cell/Cell.tsx

```diff
@@ -120,6 +120,7 @@ const Cell: React.FC<Props> = (props) => {
         onChangeNodeValue={onChangeNodeValue}
         onPaste={onPaste}
       />
+      {nodeValue?.estimated ? <div className="estimation-mark">E</div> : null}
```

Comment:
this should appear if and only if the node has estimationUuid. estimated is a deprecated property